### PR TITLE
style(Anchor): Adds new anchor variants and styles necessary for usage [GM-159] [GM-160]

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

Fleshes out finalized anchor variants and styles and adds color modes for each.

* Adds full style variants for anchors
* Color mode mapping
* Updates consumer tests to use gamut-tests
* Defaults to button if no Href specified.
* Forwardable with `.withComponent` for ReduxFirstRouterLink
* Forwards refs by default (this must be implemented separately if user applies styles to another component).

### PR Checklist

- [x] Related to designs: [Figma](https://www.figma.com/file/N6TfoPI5OA7GanX6Zl8nWj/Buttons-%26-Links?node-id=727%3A158)
- [x] Related to JIRA ticket: https://codecademy.atlassian.net/browse/GM-160 https://codecademy.atlassian.net/browse/GM-159
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.0a092c.0<br/>@codecademy/gamut-kit@0.2.11-alpha.0a092c.0<br/>@codecademy/gamut-labs@11.1.1-alpha.0a092c.0<br/>@codecademy/gamut-styles@8.3.2-alpha.0a092c.0<br/>@codecademy/gamut-tests@2.2.2-alpha.0a092c.0<br/>@codecademy/gamut@26.2.3-alpha.0a092c.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.0a092c.0<br/>@codecademy/styleguide@30.4.3-alpha.0a092c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604807ed6e0edd200be646c4--gamut-preview.netlify.app>https://604807ed6e0edd200be646c4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604807ed6e0edd200be646c4>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.509139.0<br/>@codecademy/gamut-kit@0.2.11-alpha.509139.0<br/>@codecademy/gamut-labs@11.1.1-alpha.509139.0<br/>@codecademy/gamut-styles@8.3.2-alpha.509139.0<br/>@codecademy/gamut-tests@2.2.2-alpha.509139.0<br/>@codecademy/gamut@26.2.3-alpha.509139.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.509139.0<br/>@codecademy/styleguide@30.4.3-alpha.509139.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60480dc3bed0cf2014ebdc88--gamut-preview.netlify.app>https://60480dc3bed0cf2014ebdc88--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60480dc3bed0cf2014ebdc88>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.6e71f6.0<br/>@codecademy/gamut-kit@0.2.14-alpha.6e71f6.0<br/>@codecademy/gamut-labs@11.2.2-alpha.6e71f6.0<br/>@codecademy/gamut-styles@8.3.2-alpha.6e71f6.0<br/>@codecademy/gamut-tests@2.2.2-alpha.6e71f6.0<br/>@codecademy/gamut@26.2.3-alpha.6e71f6.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.6e71f6.0<br/>@codecademy/styleguide@30.5.2-alpha.6e71f6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604a65930976be1a3570ca9d--gamut-preview.netlify.app>https://604a65930976be1a3570ca9d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604a65930976be1a3570ca9d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.c0dee3.0<br/>@codecademy/gamut-kit@0.2.14-alpha.c0dee3.0<br/>@codecademy/gamut-labs@11.2.2-alpha.c0dee3.0<br/>@codecademy/gamut-styles@8.3.2-alpha.c0dee3.0<br/>@codecademy/gamut-tests@2.2.2-alpha.c0dee3.0<br/>@codecademy/gamut@26.2.3-alpha.c0dee3.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.c0dee3.0<br/>@codecademy/styleguide@30.5.2-alpha.c0dee3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604a68f2a767d01dee672242--gamut-preview.netlify.app>https://604a68f2a767d01dee672242--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604a68f2a767d01dee672242>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.ff22fb.0<br/>@codecademy/gamut-kit@0.2.14-alpha.ff22fb.0<br/>@codecademy/gamut-labs@11.2.2-alpha.ff22fb.0<br/>@codecademy/gamut-styles@8.3.2-alpha.ff22fb.0<br/>@codecademy/gamut-tests@2.2.2-alpha.ff22fb.0<br/>@codecademy/gamut@26.2.3-alpha.ff22fb.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.ff22fb.0<br/>@codecademy/styleguide@30.5.2-alpha.ff22fb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604a6db4b58a9a1ccd0f3f53--gamut-preview.netlify.app>https://604a6db4b58a9a1ccd0f3f53--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604a6db4b58a9a1ccd0f3f53>Deploy Logs</a></p>

File: packages/styleguide/.storybook/decorators/theme.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
   createEmotionCache,
 } from '@codecademy/gamut-styles';
 
-const cache = createEmotionCache();
+const cache = createEmotionCache({ speedy: false });
```
Comment (codecaaron) [inline packages/styleguide/.storybook/decorators/theme.tsx:9]: easier debugging

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.40af0d.0<br/>@codecademy/gamut-kit@0.2.14-alpha.40af0d.0<br/>@codecademy/gamut-labs@11.2.2-alpha.40af0d.0<br/>@codecademy/gamut-styles@8.3.2-alpha.40af0d.0<br/>@codecademy/gamut-tests@2.2.2-alpha.40af0d.0<br/>@codecademy/gamut@26.2.3-alpha.40af0d.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.40af0d.0<br/>@codecademy/styleguide@30.5.2-alpha.40af0d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604a736612f7b726e0345a56--gamut-preview.netlify.app>https://604a736612f7b726e0345a56--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604a736612f7b726e0345a56>Deploy Logs</a></p>

**Review by jakemhiller — APPROVED** [review]
lgtm

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.2a7f49.0<br/>@codecademy/gamut-kit@0.2.14-alpha.2a7f49.0<br/>@codecademy/gamut-labs@11.2.2-alpha.2a7f49.0<br/>@codecademy/gamut-styles@8.3.2-alpha.2a7f49.0<br/>@codecademy/gamut-tests@2.2.2-alpha.2a7f49.0<br/>@codecademy/gamut@26.2.3-alpha.2a7f49.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.2a7f49.0<br/>@codecademy/styleguide@30.5.2-alpha.2a7f49.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604a7f476ad8b1384917f21a--gamut-preview.netlify.app>https://604a7f476ad8b1384917f21a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604a7f476ad8b1384917f21a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.397abd.0<br/>@codecademy/gamut-kit@0.2.15-alpha.397abd.0<br/>@codecademy/gamut-labs@11.2.3-alpha.397abd.0<br/>@codecademy/gamut-styles@8.3.2-alpha.397abd.0<br/>@codecademy/gamut-tests@2.2.2-alpha.397abd.0<br/>@codecademy/gamut@26.2.4-alpha.397abd.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.397abd.0<br/>@codecademy/styleguide@30.5.3-alpha.397abd.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604a809f630e0e3498a3ccfe--gamut-preview.netlify.app>https://604a809f630e0e3498a3ccfe--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604a809f630e0e3498a3ccfe>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.887225.0<br/>@codecademy/gamut-kit@0.2.15-alpha.887225.0<br/>@codecademy/gamut-labs@11.2.3-alpha.887225.0<br/>@codecademy/gamut-styles@8.3.2-alpha.887225.0<br/>@codecademy/gamut-tests@2.2.2-alpha.887225.0<br/>@codecademy/gamut@26.2.4-alpha.887225.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.887225.0<br/>@codecademy/styleguide@30.5.3-alpha.887225.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604a95cf6e6d2f4c043ad1c6--gamut-preview.netlify.app>https://604a95cf6e6d2f4c043ad1c6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604a95cf6e6d2f4c043ad1c6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.ee63f9.0<br/>@codecademy/gamut-kit@0.2.15-alpha.ee63f9.0<br/>@codecademy/gamut-labs@11.2.3-alpha.ee63f9.0<br/>@codecademy/gamut-styles@8.3.2-alpha.ee63f9.0<br/>@codecademy/gamut-tests@2.2.2-alpha.ee63f9.0<br/>@codecademy/gamut@26.2.4-alpha.ee63f9.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.ee63f9.0<br/>@codecademy/styleguide@30.5.3-alpha.ee63f9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604ac2ef514fb2120fac8e87--gamut-preview.netlify.app>https://604ac2ef514fb2120fac8e87--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604ac2ef514fb2120fac8e87>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.cf2350.0<br/>@codecademy/gamut-kit@0.2.15-alpha.cf2350.0<br/>@codecademy/gamut-labs@11.2.3-alpha.cf2350.0<br/>@codecademy/gamut-styles@8.3.2-alpha.cf2350.0<br/>@codecademy/gamut-tests@2.2.2-alpha.cf2350.0<br/>@codecademy/gamut@26.2.4-alpha.cf2350.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.cf2350.0<br/>@codecademy/styleguide@30.5.3-alpha.cf2350.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604ad6906b9cd51c4326643e--gamut-preview.netlify.app>https://604ad6906b9cd51c4326643e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604ad6906b9cd51c4326643e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.dfa63b.0<br/>@codecademy/gamut-kit@0.2.15-alpha.dfa63b.0<br/>@codecademy/gamut-labs@11.2.3-alpha.dfa63b.0<br/>@codecademy/gamut-styles@8.3.2-alpha.dfa63b.0<br/>@codecademy/gamut-tests@2.2.2-alpha.dfa63b.0<br/>@codecademy/gamut@26.2.4-alpha.dfa63b.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.dfa63b.0<br/>@codecademy/styleguide@30.5.3-alpha.dfa63b.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.15b15d.0<br/>@codecademy/gamut-kit@0.2.15-alpha.15b15d.0<br/>@codecademy/gamut-labs@11.2.3-alpha.15b15d.0<br/>@codecademy/gamut-styles@8.3.2-alpha.15b15d.0<br/>@codecademy/gamut-tests@2.2.2-alpha.15b15d.0<br/>@codecademy/gamut@26.2.4-alpha.15b15d.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.15b15d.0<br/>@codecademy/styleguide@30.5.3-alpha.15b15d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604af4170745f4457b1b6c74--gamut-preview.netlify.app>https://604af4170745f4457b1b6c74--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604af4170745f4457b1b6c74>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
🚀

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.66b2a5.0<br/>@codecademy/gamut-kit@0.2.16-alpha.66b2a5.0<br/>@codecademy/gamut-labs@11.2.4-alpha.66b2a5.0<br/>@codecademy/gamut-styles@8.3.2-alpha.66b2a5.0<br/>@codecademy/gamut-tests@2.2.2-alpha.66b2a5.0<br/>@codecademy/gamut@26.3.1-alpha.66b2a5.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.66b2a5.0<br/>@codecademy/styleguide@30.6.1-alpha.66b2a5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604b9f26370ed42f05cd5dff--gamut-preview.netlify.app>https://604b9f26370ed42f05cd5dff--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604b9f26370ed42f05cd5dff>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.b013b9.0<br/>@codecademy/gamut-kit@0.2.16-alpha.b013b9.0<br/>@codecademy/gamut-labs@11.2.4-alpha.b013b9.0<br/>@codecademy/gamut-styles@8.3.2-alpha.b013b9.0<br/>@codecademy/gamut-tests@2.2.2-alpha.b013b9.0<br/>@codecademy/gamut@26.3.1-alpha.b013b9.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.b013b9.0<br/>@codecademy/styleguide@30.6.1-alpha.b013b9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604ba13582825a297c53ebc9--gamut-preview.netlify.app>https://604ba13582825a297c53ebc9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604ba13582825a297c53ebc9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.89c02a.0<br/>@codecademy/gamut-kit@0.2.16-alpha.89c02a.0<br/>@codecademy/gamut-labs@11.2.4-alpha.89c02a.0<br/>@codecademy/gamut-styles@8.3.2-alpha.89c02a.0<br/>@codecademy/gamut-tests@2.2.2-alpha.89c02a.0<br/>@codecademy/gamut@26.3.1-alpha.89c02a.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.89c02a.0<br/>@codecademy/styleguide@30.6.1-alpha.89c02a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604ba40839bba02dad6cdd66--gamut-preview.netlify.app>https://604ba40839bba02dad6cdd66--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604ba40839bba02dad6cdd66>Deploy Logs</a></p>

File: packages/gamut-labs/src/landingPage/__tests__/Hero-test.tsx [hunk 2]
```diff
@@ -1,28 +1,29 @@
-import { fireEvent, render } from '@testing-library/react';
-import React from 'react';
+import { setupRtl } from '@codecademy/gamut-tests';
```
Comment (saghdaey) [inline packages/gamut-labs/src/landingPage/__tests__/Hero-test.tsx:1]: woot!

**Review by saghdaey — APPROVED** [review]
lgtm!! compared the storybook preview here for global header components w/ global header in production. 

(WOW FEELS GOOD TO WRITE THAT!!!!)

**Review by katiezutter — APPROVED** [review]
also LGTM from a GlobalHeader perspective!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.f68d9f.0<br/>@codecademy/gamut-kit@0.2.16-alpha.f68d9f.0<br/>@codecademy/gamut-labs@11.2.4-alpha.f68d9f.0<br/>@codecademy/gamut-styles@8.3.2-alpha.f68d9f.0<br/>@codecademy/gamut-tests@2.2.2-alpha.f68d9f.0<br/>@codecademy/gamut@26.3.1-alpha.f68d9f.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.f68d9f.0<br/>@codecademy/styleguide@30.6.1-alpha.f68d9f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604ee45e648d6c55889cc8e3--gamut-preview.netlify.app>https://604ee45e648d6c55889cc8e3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604ee45e648d6c55889cc8e3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.8e228e.0<br/>@codecademy/gamut-kit@0.2.16-alpha.8e228e.0<br/>@codecademy/gamut-labs@11.2.4-alpha.8e228e.0<br/>@codecademy/gamut-styles@8.3.2-alpha.8e228e.0<br/>@codecademy/gamut-tests@2.2.2-alpha.8e228e.0<br/>@codecademy/gamut@26.3.1-alpha.8e228e.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.8e228e.0<br/>@codecademy/styleguide@30.6.1-alpha.8e228e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://604ef159a9112e64694fa0b0--gamut-preview.netlify.app>https://604ef159a9112e64694fa0b0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/604ef159a9112e64694fa0b0>Deploy Logs</a></p>

File: packages/gamut-labs/src/landingPage/__tests__/Hero-test.tsx [hunk 3]
```diff
@@ -32,43 +33,41 @@ describe('PageHero', () => {
...
+      cta: { text: 'cta', href: 'https://codecademy.com' },
+    });
     wrapper.getByText('cta');
   });
 
   it('should render an img when media prop with type="image" is provided', () => {
-    const wrapper = render(
-      <PageHero media={{ type: 'image', src: 'test', alt: 'alt' }} />
-    );
+    const { view: wrapper } = renderView({
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/landingPage/__tests__/Hero-test.tsx:43]: Wanna just call these `view` and be done with it? 😄

File: packages/gamut/src/Anchor/index.tsx [hunk 4]
```diff
@@ -1,25 +1,172 @@
...
+  background: none;
+  box-shadow: none;
+  border: none;
+  padding: 0;
+  font-size: inherit;
 `;
 
+const AnchorElement = forwardRef<LinkElements, ForwardedProps>(
+  ({ href, disabled, children, as, ...rest }, ref) => {
+    if (!href || href.length === 0) {
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/Anchor/index.tsx:92]: ⛳ ✨ 

```suggestion
    if (!href?.length) {
```

**Review by JoshuaKGoldberg — APPROVED** [review]
What a relief to finally have these standardized! Thanks!!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.b236d6.0<br/>@codecademy/gamut-kit@0.2.20-alpha.b236d6.0<br/>@codecademy/gamut-labs@12.0.0-alpha.b236d6.0<br/>@codecademy/gamut-styles@8.3.2-alpha.b236d6.0<br/>@codecademy/gamut-tests@2.2.2-alpha.b236d6.0<br/>@codecademy/gamut@27.0.0-alpha.b236d6.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.b236d6.0<br/>@codecademy/styleguide@31.0.0-alpha.b236d6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6050bdc733eed3c62076a5a3--gamut-preview.netlify.app>https://6050bdc733eed3c62076a5a3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6050bdc733eed3c62076a5a3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-illustrations@0.8.1-alpha.c33def.0<br/>@codecademy/gamut-kit@0.2.21-alpha.c33def.0<br/>@codecademy/gamut-labs@12.0.1-alpha.c33def.0<br/>@codecademy/gamut-styles@8.3.2-alpha.c33def.0<br/>@codecademy/gamut-tests@2.2.2-alpha.c33def.0<br/>@codecademy/gamut@27.0.1-alpha.c33def.0<br/>@codecademy/markdown-overrides@0.5.9-alpha.c33def.0<br/>@codecademy/styleguide@31.0.1-alpha.c33def.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6050bf5b71e84dc83c72701f--gamut-preview.netlify.app>https://6050bf5b71e84dc83c72701f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6050bf5b71e84dc83c72701f>Deploy Logs</a></p>

**Pull request merged**